client/blocks/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -19,8 +19,8 @@ import videoImage from 'assets/images/illustrations/video-hosting.svg';
 function getDescription( plan, translate ) {
 	if ( isWpComBusinessPlan( plan ) ) {
 		return translate(
-			'Enrich your posts and pages with video or audio. Upload as much media as you want, ' +
-				'directly to your site — the Business Plan has unlimited storage.'
+			'Enrich your site and boost engagement with video and audio. Your Business Plan includes 200GB ' +
```

Comment:
Let's change this to include a space between the number and the measurement. `200 GB`